.buildkite/build_pr_pipeline.yml

```diff
@@ -6,7 +6,16 @@ steps:
     command: ".buildkite/scripts/cancel_running_pr.sh || true"
   - key: "build-pr-setup"
     label: "setup"
-    command: ".buildkite/scripts/build_pr_commit_status.sh pending"
+    command: |
+      curl -s -L \
+        -X POST \
+        -H "Accept: application/vnd.github+json" \
+        -H "Authorization: Bearer $${VAULT_GITHUB_TOKEN}" \
```

Comment:
avoid interpolation and therefore exposing credentials

Reply:
This was an interesting read in Slack